packages/core/src/signals/src/signal.ts

```diff
@@ -137,6 +153,7 @@ export function signal<T>(initialValue: T, options?: CreateSignalOptions<T>): Wr
                      set: signalNode.set.bind(signalNode),
                      update: signalNode.update.bind(signalNode),
                      mutate: signalNode.mutate.bind(signalNode),
+                     asReadonly: signalNode.asReadonly.bind(signalNode)
```

Comment:
Is it worth the code size to memoize things here? We could instead do:
```suggestion
                     asReadonly: () => createSignalFromFunction(signalNode, signalNode.signal.bind(signalNode)),
```